pkgs/build-support/testers/default.nix

```diff
@@ -97,17 +97,23 @@
   # See doc/builders/testers.chapter.md or
   # https://nixos.org/manual/nixpkgs/unstable/#tester-runNixOSTest
   runNixOSTest =
-    let nixos = import ../../../nixos/lib {
-      inherit lib;
-    };
+    let
+      toGuest = builtins.replaceStrings [ "darwin" ] [ "linux" ];
+      guestPkgs =
+        if (!pkgs.stdenv.hostPlatform.isDarwin)
+          then pkgs
+          else import pkgs.path { system = toGuest pkgs.system; };
```

Comment:
Would be nice to have `pkgsLinux`, like we have `pkgsStatic` or `pkgsCross.<x>` (although this wouldn't be cross; just a related platform that's built natively)

Reply:
Or `pkgsNativePlatform.linux` or something, with _native_ referring both to not being cross and being able to run on the same hardware that runs `pkgs`.

Reply:
True. Although this would blow up this PR... can we discuss this offline after this PR how to improve it?